pushapkscript/test/test_googleplay.py

```diff
@@ -134,50 +170,44 @@ def test_should_commit_transaction(self):
         del self.context.task['payload']['commit']
         self.assertFalse(should_commit_transaction(self.context))
 
-    def test_get_google_play_strings_path(self):
-        self.assertEqual(
-            get_google_play_strings_path({'someTaskId': ['/path/to/public/google_play_strings.json']}, {}),
-            '/path/to/public/google_play_strings.json'
-        )
-        self.assertEqual(
-            get_google_play_strings_path(
-                {'apkTaskId': ['/path/to/public/build/target.apk']},
-                {'gpStringTaskId': ['public/google_play_strings.json']}
-            ),
-            None
-        )
-        # Error cases checked in subfunctions
-
-    def test_check_google_play_string_is_the_only_failed_task(self):
-        with self.assertRaises(TaskVerificationError):
-            _check_google_play_string_is_the_only_failed_task({
-                'apkTaskId': ['/path/to/public/build/target.apk'],
-                'gpStringTaskId': ['public/chainOfTrust.json.asc', 'public/google_play_strings.json']
-            })
-
-        with self.assertRaises(TaskVerificationError):
-            _check_google_play_string_is_the_only_failed_task({
-                'missingJsonTaskId': ['public/chainOfTrust.json.asc']
-            })
-
-    def test_find_unique_google_play_strings_file_in_dict(self):
-        self.assertEqual(
-            _find_unique_google_play_strings_file_in_dict({
-                'apkTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/build/target.apk'],
-                'someTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/google_play_strings.json'],
-            }),
-            '/path/to/public/google_play_strings.json'
-        )
 
-        with self.assertRaises(TaskVerificationError):
-            _find_unique_google_play_strings_file_in_dict({
-                'apkTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/build/target.apk'],
-                'someTaskId': ['public/chainOfTrust.json.asc'],
-            })
-
-        with self.assertRaises(TaskVerificationError):
-            _find_unique_google_play_strings_file_in_dict({
-                'apkTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/build/target.apk'],
-                'someTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/google_play_strings.json'],
-                'someOtherTaskId': ['public/chainOfTrust.json.asc', '/path/to/public/google_play_strings.json'],
-            })
+def test_get_google_play_strings_path():
```

Comment:
These didn't need to be part of the class, so I pulled them out so I wouldn't need to add an empty `_` for `mock_push_apk`